specification/cognitiveservices/data-plane/SpeakerRecognition/Verification/stable/2021-09-05/Verification.json

```diff
@@ -0,0 +1,1148 @@
+      "enum": [
+        "Enrolling",
+        "Training",
+        "Enrolled"
+      ],
+      "x-ms-enum": {
+        "name": "TrainingStatusType",
+        "modelAsString": false
+      },
+      "example": "Enrolling"
+    },
+    "CreatedDateTime": {
+      "type": "string",
+      "format": "date-time",
+      "description": "Profile creation datetime.",
+      "example": "2015-04-23T18:25:43.41Z"
+    },
+    "LastUpdatedDateTime": {
+      "type": "string",
+      "format": "date-time",
+      "description": "Last datetime when the profile was updated.",
+      "example": "2015-04-23T19:34:51.52Z"
+    },
+    "EnrollmentsCount": {
+      "type": "integer",
+      "description": "Number of enrollment audios accepted for this profile.",
+      "example": 1
+    },
+    "EnrollmentsLengthInSec": {
+      "type": "number",
+      "description": "Total length of enrollment audios accepted for this profile in seconds.",
+      "example": 1.83
+    },
+    "EnrollmentsSpeechLengthInSec": {
+      "type": "number",
+      "description": "Summation of pure speech (which is the amount of audio after removing silence and non-speech segments) across all profile enrollments in seconds.",
+      "example": 1.35
+    },
+    "RemainingEnrollmentsCount": {
+      "type": "integer",
+      "description": "Number of enrollment audios needed to complete profile enrollment.",
+      "example": 2
+    },
+    "RemainingEnrollmentsSpeechLengthInSec": {
+      "type": "number",
+      "description": "Amount of pure speech (which is the amount of audio after removing silence and non-speech segments) needed to complete profile enrollment in seconds.",
+      "example": 18.65
+    },
+    "ModelVersion": {
+      "type": "string",
+      "format": "date",
+      "description": "Date specifying the model assigned to this profile. Format is yyyy-mm-dd. If profile has no enrollments, this value will be empty.",
+      "example": "2019-12-05"
+    },
+    "PassPhrase": {
+      "type": "string",
+      "description": "Passphrase associated with this enrollment.",
+      "example": "my voice is my passport verify me"
+    },
+    "ActivationPhrase": {
+      "type": "string",
+      "description": "Activation phrases available to activate a profile.",
+      "example": "this is my activation phrase to identify me"
+    },
+    "LocaleInfo": {
+      "description": "Speaker profile locale",
+      "properties": {
+        "locale": {
+          "$ref": "#/definitions/Locale"
+        }
+      }
+    },
+    "TdProfileInfoList": {
+      "description": "Text-Dependent Speaker profile info list",
+      "properties": {
+        "profiles": {
```

Comment:
The Azure API Guidelines say that this property should be named `value`. And whether or not you change the name, you should list this property as `required`.

Reply:
done